pkg/token/kubernetes_exec_infoIsEmpty_test.go

```diff
@@ -0,0 +1,36 @@
+package token
+
+import (
+	"os"
+	"testing"
+)
+
+func TestKUBERNETES_EXEC_INFOIsempty(t *testing.T) {
+	testData := []struct {
+		name            string
+		execInfoEnvTest string
+		options         Options
+	}{
+		{
+			name:            "KUBERNETES_EXEC_INFO is empty",
+			execInfoEnvTest: "",
+			options: Options{
+				LoginMethod: DeviceCodeLogin,
+				ClientID:    "clientID",
+				ServerID:    "serverID",
+				TenantID:    "tenantID",
+			},
+		},
+	}
+
+	for _, data := range testData {
```

Comment:
i can you can simply update existing unit tests?

Reply:
Done in the most recent commit 3c28aca and please check it out. Thanks!